libs/core/src/components/pds-button/docs/pds-button.mdx

```diff
@@ -143,6 +120,27 @@ Destructive button variant, intended to be used with destructive actions, such a
   <pds-button variant="destructive" icon="trash">Icon</pds-button>
 </DocCanvas>
 
+### Disclosure
+
+Disclosure button variant, intended to be used as a trigger to show or hide content elsewhere on the page.
+
+> **Note:** Icon cannot be customized.
+
+<DocCanvas client:only
+  mdxSource={{
+    react: `
+    <PdsButton variant="disclosure">Disclosure</PdsButton>
+    <PdsButton variant="disclosure" disabled="true">Disabled</PdsButton>
+    `,
+    webComponent: `
+    <pds-button variant="disclosure">Disclosure</pds-button>
+    <pds-button variant="disclosure" disabled="true">Disabled</pds-button>
+    `
+}}>
+  <pds-button variant="disclosure">Disclosure</pds-button>
+  <pds-button variant="disclosure" disabled="true">Disabled</pds-button>
+</DocCanvas>
+
```

Comment:
Looking at this again, what makes this different than the newly added "CTA" Button? Other than the `href`

Reply:
I just moved the ordering of this one, since it set in between the more similar variant types. Not sure I understand the question though.